src/caustics/lenses/epl.py

```diff
@@ -46,10 +47,12 @@ class EPL(ThinLens):
         This is the orientation of the lens on the sky,
         typically given as an angle measured counter-clockwise
         from some reference direction.
+    *Unit: radians*
```

Comment:
The spacing doesn't match the rest of the usages.
There are missing units above.